### ISSUE TITLE
Expose Grid renderers for DG native data types

### ISSUE THREAD

**Issue by dgosi** [datagrok-ai/public#340]
Please add the ability to retrieve renderers (instances of GridCellRenderer class) for any column in GridViewer

**Comment by alex-aprm** [comment]
@skalkin please, take a look, it's quite urgent

**Comment by skalkin** [comment]
Implemented in v0.140

```
// Reusing core grid renderers

let view = grok.shell.addTableView(grok.data.demo.molecules());
view.dataFrame.col('age').colors.setLinear();
let canvas = ui.canvas(300, 200);

view.grid.onCurrentCellChanged.subscribe((gridCell) =>
  gridCell.renderer.render(canvas.getContext('2d'), 20, 20, 200, 100, gridCell, gridCell.style));

ui.dialog()
  .add('Click on a cell to render it in the canvas below')
  .add(canvas)
  .show();
```

![image](https://user-images.githubusercontent.com/943403/160000131-5f7e07aa-0f56-4e80-a7a6-38a3d19b28c0.png)
